tests/models_tests/test_scan.py

```diff
@@ -202,9 +202,9 @@ def test_get_spatial_resolution_from_dicom(self):
     #     expected = TWO_DIM_TEST_SERIES["spatial_resolution"]
     #     self.assertListEqual(result, expected)
 
-    def test_infer_sequence_type_from_dicom_returns_none(self):
-        result = self.scan.infer_sequence_type_from_dicom()
-        self.assertIsNone(result)
+    # def test_infer_sequence_type_from_dicom_returns_none(self):
+    #     result = self.scan.infer_sequence_type_from_dicom()
+    #     self.assertIsNone(result)
 
```

Comment:
This should still be tested. Perhaps is is better to create the SequenceType instances only in the relevant tests, or perhaps remove them for this particular test.

Reply:
removed the population of SequenceType table from setUpTestData, moved it to test_sequence_type. The test is back and passes.